src/app/DTT/address/index.js

```diff
@@ -1,11 +1,10 @@
 import { Read } from 'app/DTT/text'
-import GooglePlacesAutocomplete from 'react-google-places-autocomplete'
-
-const Write = ({ onSendAnswer, data }) => <GooglePlacesAutocomplete />
-
+import GetAddress from './get_address'
+const Write = ({ onSendAnswer, data }) => {
+  return <GetAddress onSendAnswer={onSendAnswer} data={data} />
```

Comment:
Having a function component start with "Get" is strange, starting with a verb is usually reserved for functions that return what comes after.

We should probably call this AddressPicker which is more common for a component